library/core/src/ptr/const_ptr.rs

```diff
@@ -2248,5 +2248,45 @@ mod verify {
     generate_const_byte_offset_from_slice_harness!(i64, check_const_byte_offset_from_i64_slice);
     generate_const_byte_offset_from_slice_harness!(i128, check_const_byte_offset_from_i128_slice);
     generate_const_byte_offset_from_slice_harness!(isize, check_const_byte_offset_from_isize_slice);
-    
+
+    trait TestTrait {}
+
+    struct TestStruct {
```

Comment:
Instead of implementing `Arbitrary` trait for `TestStruct`, you can automatically derive the trait for the struct as all of its underlying fields (`value`) implement the trait.

Use the `derive` macro on top of the struct to do this:

```suggestion
    #[cfg_attr(kani, derive(kani::Arbitrary))]
    struct TestStruct {
```